### DISCUSSION TITLE
Any chance to programmatically hide files?

### DISCUSSION THREAD

**Discussion by mariomui** [Eldritch-Oliver/file-hider#34] [Ideas]
I'd love to hide all the files in my vault that begin with an underscore.

**Comment by Eldritch-Oliver** [comment]
This is something that I'd like to do, though it will likely take the shape of RegEx pattern matching. But if you don't think that that would suit your needs I'm more than open to hearing suggestions on other requirements.

**Reply by mariomui** [reply]
I don't mind regex. In fact that is the preferable option. 
I would suggest other features but i sortah believe in incremental building. 
> * feat 1: regex
> * feat 2: saved regex for non power users (so they options like "hide filest that start with") that utilizes regex engine
> * feat 3: Differentiate between file and folder.

**Reply by Eldritch-Oliver** [reply]
Okay yeah, I'm looking at getting RegEx added to the system, though I'm wanting to try and improve the way the plugin is working before digging into that completely as right now the way it manages the hidden files/folders is quite awful.
